GitHubAPI/Model/JsonResponse/GithubUserJsonResponse.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHubAPI.Model.JsonResponse
+{
+    class GithubUserJsonResponse
+    {
+        public int id { get; set; }
```

Comment:
Change Property naming to Pascal Case -> follow C# naming conversion. Add JsonProperty annotation to parse response properly